helm_deploy/hmpps-interventions-service/templates/_envs.tpl

```diff
@@ -34,6 +34,12 @@ env:
   - name: COMMUNITYAPI_CONTACTNOTIFICATIONCONTEXT_TEAMCODE
     value: "{{ .Values.env.COMMUNITYAPI_CONTACTNOTIFICATIONCONTEXT_TEAMCODE }}"
 
+  - name: APPLICATION_INSIGHTS_IKEY
+    valueFrom:
+      secretKeyRef:
+        name: {{ template "app.name" . }}
+        key: APPINSIGHTS_INSTRUMENTATIONKEY
```

Comment:
🙋‍♂️ we essentially have key/value pairs namespaced under the secret name, so if you'd like we can call it

[secret name] `application-insights` &rarr; [key inside secret] `instrumentation-key`
vs current
[secret name] `application-insights-key` &rarr; [key inside secret] `APPINSIGHTS_INSTRUMENTATIONKEY `

absolutely not a blocker, just an opportunity to lean on the namespace/key/value semantics